### PR TITLE
Revamp kinto-admin install/setup/upgrade procedure.

### PR DESCRIPTION
**Pull request by n1k0**
This is mostly to get feedback but I think that's a simple yet efficient approach.

/cc @leplatrem @Natim

### CODE REVIEW

File: kinto/plugins/admin/public/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>React App</title>
```
Comment (n1k0) [inline kinto/plugins/admin/public/index.html]: No.

File: kinto/plugins/admin/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "_admin",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "http://placeholder/admin/",
```
Comment (n1k0) [inline kinto/plugins/admin/package.json]: This is a trick to set the public asset root path to `/admin/`.

File: kinto/plugins/admin/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "devDependencies": {
+    "react-scripts": "0.7.0"
+  },
+  "dependencies": {
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2"
```
Comment (n1k0) [inline kinto/plugins/admin/package.json]: When kinto-admin v1.4.1 is out, we need to add `"kinto-admin": "1.4.1"` here.

File: kinto/plugins/admin/package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
```
Comment (n1k0) [inline kinto/plugins/admin/package.json]: We could remove this and a few other items.

File: kinto/plugins/admin/package.json (hunk 2 above)
Comment (Natim) [inline kinto/plugins/admin/package.json]: It should be `/v1/admin/` see #886

**Comment by n1k0** [comment]
Closing in favor of #886

**Pull request closed without merging**